UnityProject/Assets/Scripts/PlayGroups/Input/InputController.cs

```diff
@@ -80,25 +80,30 @@ private void CheckClick()
 
 		private void CheckAltClick()
 		{
-			if (UnityEngine.Input.GetMouseButtonDown(0) && (UnityEngine.Input.GetKey(KeyCode.LeftAlt) || UnityEngine.Input.GetKey(KeyCode.RightAlt)))
+			if ( UnityEngine.Input.GetMouseButtonDown(0) && ( UnityEngine.Input.GetKey(KeyCode.LeftAlt) || UnityEngine.Input.GetKey(KeyCode.RightAlt) ) )
 			{
 				//Check for items on the clicked possition, and display them in the Item List Tab, if they're in reach
 				Vector3 position = Camera.main.ScreenToWorldPoint(UnityEngine.Input.mousePosition);
 				position.z = 0f;
-				if (PlayerManager.LocalPlayerScript.IsInReach(position))
+				if ( PlayerManager.LocalPlayerScript.IsInReach(position) )
 				{
 					List<GameObject> objects = UITileList.GetItemsAtPosition(position);
 					LayerTile tile = UITileList.GetTileAtPosition(position);
-					ControlTabs.ShowItemListTab(objects, tile, position);
+					if ( tile )
+					{
+						ControlTabs.ShowItemListTab(objects, tile, position);
+					}
 				}
+				Debug.LogFormat($"clicked position:{Vector3Int.RoundToInt(position)}");

```

Comment:
Does this still need to be here?

Reply:
It's actually a handy feature that helped me a lot when debugging floating

Reply:
So shouldn't it be commented for debugging?

Reply:
Ah, that's the log string, yeah, that one's supposed to be commented